calvinpete/Glokeeper-backend
# #181851590 Setup and deploy base application to heroku

#### What does this PR do?
This PR is to set up and deploy base application to Heroku.

#### Description of the task to be completed.
Setup and deploy base application to Heroku so that other users can access the application.

#### How should this be manually tested?
- First things first, fetch this branch by running `git fetch origin  ch-deploy-heroku-181851590`
- Check into this branch by running `git checkout --  ch-deploy-heroku-181851590`
- Activate your virtual environment by running `source venv/bin/activate`
- Install dependencies by running `pip install -r requirements.txt`
- View live `staging app` [here](https://will-gen-app-staging.herokuapp.com/admin/)

To run locally, you need to have a `heroku-cli` installed on your machine
[Link on how to install `heroku cli](https://devcenter.heroku.com/articles/heroku-cli)

- run local Heroku local server using` heroku local web`
- navigate to `localhost:8000/admin/` in your web browser

#### Relevant stories
[#181851590](https://www.pivotaltracker.com/story/show/181851590)
